### PR TITLE
Added a clarification to return the new value from the setter.

### PR DESCRIPTION
**Pull request by lindem**
Hi all, 

I feel this sentence can prevent others to miss (like we did) :-) actually returning the value after doing what they intended to do in the setter function.

Cheers

### CODE REVIEW

File: source/localizable/object-model/computed-properties.md [hunk 1]
```diff
@@ -91,7 +91,10 @@ Setting any dependent property will propagate changes through any computed prope
 
 ### Setting Computed Properties
 
-You can also define what Ember should do when setting a computed property. If you try to set a computed property, it will be invoked with the key (property name), and the value you want to set it to.
+You can also define what Ember should do when setting a computed property. If
```
Comment (locks) [inline source/localizable/object-model/computed-properties.md]: please break the line at the period

File: source/localizable/object-model/computed-properties.md [hunk 2]
```diff
@@ -91,7 +91,10 @@ Setting any dependent property will propagate changes through any computed prope
...
+ you try to set a computed property, it will be invoked with the key 
+ (property name), and the value you want to set it to. You must return the
```
Comment (locks) [inline source/localizable/object-model/computed-properties.md]: see above

File: source/localizable/object-model/computed-properties.md [hunk 3]
```diff
@@ -91,7 +91,8 @@ Setting any dependent property will propagate changes through any computed prope
 
 ### Setting Computed Properties
 
-You can also define what Ember should do when setting a computed property. If you try to set a computed property, it will be invoked with the key (property name), and the value you want to set it to.
+You can also define what Ember should do when setting a computed property. If you try to set a computed property, it will be invoked with the key (property name), and the value you want to set it to.
```
Comment (locks) [inline source/localizable/object-model/computed-properties.md]: could you insert a newline after the period, please?

**Comment by lindem** [comment]
sorry, I'm not at the office right now :-) didn't see it. There you are.

**Comment by locks** [comment]
Thanks for your contribution!

@homu r+

**Pull request merged**